admin-dev/themes/default/js/bundle/product/catalog.js

```diff
@@ -71,8 +71,9 @@ $(document).ready(function() {
 	 * Sortable case when ordered by position ASC
 	 */
 
-	$("body").on("mousedown", "tbody.sortable [data-uniturl]", function () {
-		$(this).find('input:checkbox[name="bulk_action_selected_products[]"]').attr("checked", true);
+	$("body").on("mousedown", "tbody.sortable [data-uniturl] td.placeholder", function () {
+	    const trParent = $(this).closest('tr');
```

Comment:
No need to use const here, you never reuse the variable.
Watch out with indentation, sometimes 2, 4 and even 6 :)